src/components/templates/ConversationSpace/ConversationSpace.scss

```diff
@@ -2,10 +2,8 @@
 
 // @debt Are all of the jazzbar / emoji related styles actually needed/used here?
 .conversation-space-container {
-  margin: 40px 40px 0 40px;
-
   .userlist-container {
-    margin-left: 40px;
+    margin-left: 80px;
   }
```

Comment:
What are the changes here based on?

Reply:
I had to move the "back" button into the conversation-space-container, so having a margin is not useful. I tried to copy these from the jazzbar

Reply:
Makes sense. If it looks fine then feel free to mark this resolved.